File: lesson4/c_c++/homework/homework4_22.cpp
//: lesson4/c_c++/homework/homewor4_22.cpp
#include <iostream>

int main() {
    int numb_1;
    std::cout << "Введите число => ";
    std::cin >> numb_1;

    int count = 1;
    while (count != 10) {
        int numb_2;
        std::cout << "Введите число => ";
        std::cin >> numb_2;
        numb_1 += numb_2;
        count++;
    }
    std::cout << "sum_numb_1 => " << numb_1;
    std::cout << " count => " << count << '\n';
    return 0;
}

File: lesson4/python3/homework/homework4_22.py
#: lesson4/python3/homework/homework4_22.py

numb_1 = int(input("Введите число => "))
count_1 = 1
while count_1 != 10:
    numb_2 = int(input("Введите число => "))
    numb_1 += numb_2
    count_1 += 1
print("numb_1 =>", numb_1, "count_1 =>", count_1)

